Test/WebTestCase.php

```diff
@@ -119,7 +119,19 @@ protected function runCommand($name, array $params = array(), $reuseKernel = fal
         $input->setInteractive(false);
 
         $fp = fopen('php://temp/maxmemory:'.$this->maxMemory, 'r+');
-        $output = new StreamOutput($fp);
+
+        $commandVerbosity = StreamOutput::VERBOSITY_NORMAL;
+        if ($this->getContainer()->hasParameter('liip_functional_test.command_verbosity')) {
```

Comment:
That's a good idea. But we could even have both.
Could you guide me on how to set new properties in that case? I am not sure were to inject those properties.
Thanks